EVENTS.md

```diff
@@ -35,21 +35,85 @@ TODO: show event structure.
 
 TODO: contrast flattened/unflattened events
 
-### Standard Events in the SDK
+### Standard Events in the Cosmos-SDK
+The Cosmos-SDK emits events of type `sdk.EventTypeMessage` for every message that contains the `sender` (actor) and `module` field (which seems to be optional unfortunately)
+For example in the **staking** [msg_server.go](https://github.com/cosmos/cosmos-sdk/blob/v0.42.9/x/staking/keeper/msg_server.go#L106-L117):
+```go
+sdk.NewEvent(
+    types.EventTypeCreateValidator,
+    sdk.NewAttribute(types.AttributeKeyValidator, msg.ValidatorAddress),
+    sdk.NewAttribute(sdk.AttributeKeyAmount, msg.Value.Amount.String()),
+)
+```
+
+They also emit one or more other custom type event in the module that describe the operation executed with more details or metadata.
+```go
+sdk.NewEvent(
```

Comment:
This is the same as above? Did you intend to link something else?

Reply:
ja, first example should have been the message type. sorry